docs/src/index.md

```diff
@@ -10,13 +10,44 @@ Suppose that you have a Julia package `Foo.jl` which wants to use a compiled `li
 
-While constructing your own `build_tarballs.jl` is certainly possible, `BinaryBuilder.jl` supports a more interactive method for building the binary dependencies and capturing the commands used to build it into a `build_tarballs.jl` file: the Wizard interface.  To launch it, run `BinaryBuilder.run_wizard()`, and follow the instructions on-screen.
+```julia
+using BinaryBuilder
+state = BinaryBuilder.run_wizard()
+```
+
+and follow the instructions on-screen.  You can watch an [asciinema demo](https://asciinema.org/a/304105) of the use of the wizard.
+
+### Manually create or edit `build_tarballs.jl`
+
+The wizard is a great tool, especially to get started with BinaryBuilder and create your first simple recipes for new packages.  However, it lacks the full control of all options you can use in a `build_tarballs.jl` script.  To generate this file (explained in greater detail in [Building Packages](building.md)), one can clone `Yggdrasil`, copy an existing build recipe, modify it, and submit a new pull request.  Manually editing the `build_tarballs.jl` script is also the recommended way when you want to update an existing recipe, rather then starting from scratch with the wizard.
+
+The `build_tarballs.jl` script can be used as a command line utility, it takes a few options and as argument the list of triplets of the targets.  You can find more information about the syntax of the script in the [Command Line](@ref) section or by running
+```
+julia build_tarballs.jl --help
+```
+You can build the tarballs with
+```
+julia build_tarballs.jl --debug --verbose
+```
+The `--debug` option will drop you into the BinaryBuilder interactive shell if an error occurs.  If the build fails, after finding out the steps needed to fix the build you have to manually update the script in `build_tarballs.jl`.  You should run again the above command to make sure that everything is actually working.
+
+Since `build_tarballs.jl` takes as argument the comma-separated list of [triplets](@ref Platforms) for which to build the tarballs, you can select only a few of them.  For example, with
+```
+julia build_tarballs.jl --debug --verbose aarch64-linux-musl,arm-linux-musleabihf
+```
+you'll run the build script only for the `aarch64-linux-musl` and `arm-linux-musleabihf` target platforms.
+
+If you decide to use this workflow, however, you will need to manually open pull requests for [Yggdrasil](https://github.com/JuliaPackaging/Yggdrasil/).
+
+### GitHub Codespaces
+
+If you already have access to the [GitHub Codespaces](https://github.com/features/codespaces) service, you can use use BinaryBuilder and all the workflows described above in your browser or with Visual Studio Code, on any operating system, including those not natively supported by the package!  Head to [Yggdrasil](https://github.com/JuliaPackaging/Yggdrasil/) and create a new Codespace.
```

Comment:
I don't know if we want to be more extensive here, for example with screenshots, but I feel like this page is already getting too long.